server/fleet/vpp.go

```diff
@@ -16,6 +16,7 @@ type VPPAppID struct {
 type VPPAppTeam struct {
 	VPPAppID
 
+	ID          uint `db:"id" json:"-"`
```

Comment:
Thinking this should be AppTeamID or similar to make it super clear that this is _not_, for example, the adam_id or the like.